tests/test_datanode.py

```diff
@@ -76,7 +76,7 @@ def test_read_datanode(client, default_df_datanode):
         config_mock.side_effect = [default_df_datanode]
         # without operators
         datanodes_url = url_for("api.datanode_reader", datanode_id="foo")
-        rep = client.get(datanodes_url)
+        rep = client.get(datanodes_url, json={})
```

Comment:
I don't understand why add the json param to this function.

Reply:
Else it failed :/